google/cloud/bigtable/table_admin_test.cc

```diff
@@ -56,8 +56,8 @@ class TableAdminTester {
     return admin.background_threads_;
   }
 
-  static Options Policies(bigtable::TableAdmin const& admin) {
-    return admin.policies_;
+  static Options Opts(bigtable::TableAdmin const& admin) {
```

Comment:
Can we use `Options()`?

Reply:
No! :'(

Reply:
Why not?  It conflicts with the name?  Can you say:

```cc
static ::google::cloud::Options Options(bigtable::TableAdmin const& admin) ...
```

Or can you rename the function to `ExtractOptions()`?

Reply:
`::google::cloud::Options Options(...)` works.

I clearly did not put up a good fight against the compiler.

Reply:
As should just `cloud::Options`, yes?  If we depend on namespace lookup at all (as we do), then we might as well use it I reckon.

Reply:
I used `::google::cloud::Options` for no particular reason.

PTAL